### PR TITLE
Updated link

### LINKED ISSUE #54: Update README file
Currently, the README.md still links [Swift JIRA](https://bugs.swift.org/) for bug report.

We should change it to https://github.com/apple/swift-markdown/issues

### PR DESCRIPTION
**Pull request by PrajwalBorkar**
Bug/issue #, if applicable: 
Close #54 
## Summary
Updated bug report link
_Provide a description of what your PR addresses, explaining the expected user experience. 
Also, provide an overview of your implementation._

## Dependencies

_Describe any dependencies this PR might have, such as an associated branch in another repository._

## Testing

_Describe how a reviewer can test the functionality of your PR. Provide test content to test with if
applicable._

Steps:
1. _Provide setup instructions._
2. _Explain in detail how the functionality can be tested._

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] Added tests
- [ ] Ran the `./bin/test` script and it succeeded
- [ ] Updated documentation if necessary

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ for more detailed information about the library.
 
 ### Submitting a Bug Report
 
-Swift Markdown tracks all bug reports with [Swift JIRA](https://bugs.swift.org/).
+Swift Markdown tracks all bug reports with [Swift JIRA](https://github.com/apple/swift-markdown/issues).
```
Comment (Kyle-Ye) [inline README.md:46]: The text of the link should also be changed accordingly. "JIRA" is used for "bugs.swift.org" website.

Also, we should update the link and text in "Submitting a Feature Request" too.

You can refer to the PR below for help.

https://github.com/apple/swift-docc/commit/f60baf619e620198d37839ca59c89697ad29a113#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

**Review by Kyle-Ye — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PrajwalBorkar) [reply README.md]: Made the changes please review.

File: README.md [hunk 2]
```diff
@@ -43,16 +43,15 @@ for more detailed information about the library.
...
 You can use the "Swift-Markdown" component for issues and feature requests specific to Swift Markdown.
 When you submit a bug report we ask that you follow the
 Swift [Bug Reporting](https://swift.org/contributing/#reporting-bugs) guidelines
 and provide as many details as possible.
 
 ### Submitting a Feature Request
 
-For feature requests, please feel free to create an issue
-on [Swift JIRA](https://bugs.swift.org/) with the `New Feature` type
+For feature requests, please feel free to file a [GitHub issue](https://github.com/apple/swift-markdown/issues/new?template=FEATURE_REQUEST.md)
```
Comment (Kyle-Ye) [inline README.md:54]: Currently, swift-markdown repo does not have issue templete.

```suggestion
For feature requests, please feel free to file a [GitHub issue](https://github.com/apple/swift-markdown/issues/new)
```

Comment (Kyle-Ye) [reply README.md:54]: Also would you mind using force push to commit only 1 commit?

Comment (PrajwalBorkar) [reply README.md]: I have not cloned to the local repo so I don't think I can do it.  If any solution please mention :)

**Comment by Kyle-Ye** [comment]
@swift-ci please test

**Review by Kyle-Ye — APPROVED** [review]

**Comment by PrajwalBorkar** [comment]
Thanks :)

**Comment by Kyle-Ye** [comment]
> Thanks :)

Thanks for your contribution. 🎉

**Pull request merged**